### ISSUE TITLE
Signs

### ISSUE THREAD

**Issue by bralvesantos** [MysticCity/MinigamesAPI#155]
I ask if they can add <motd> to the signs. 

Thanks, Bruno.

**Comment by mepeisen** [comment]
What exactly are you thinking of? Displaying the motd of another server within bungee networks?
Please give a concrete example.

**Comment by bralvesantos** [comment]
Yeah, for example, In the sign on lobby, it will appear the motd of the server arena. eg.

    '0': '&a<motd>' - This indicates the motd of arena 
    '1': '&a<arena>' - This is the name of the map
    '2': '&a<count>/<maxcount>'
    '3': '&a[]'
